sonar-plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -244,6 +244,7 @@ public boolean isInternal() {
 
   class NewAction {
     private final String key;
+    private final String PAGE_PARAM_DESCRIPTION = "1-based page number";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this final field static too. ([squid:S1170](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1170))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWGfL87_R7xclXvFfE0q&open=AWGfL87_R7xclXvFfE0q)

[ISSUE_KEY]: AWGfL87_R7xclXvFfE0q